Formula/spack.rb

```diff
@@ -4,6 +4,7 @@ class Spack < Formula
   url "https://github.com/spack/spack/archive/v0.16.2.tar.gz"
```

Comment:
Let's add a comment above this line that it can be switched back to `python@3.9` at `v0.16.3`.